cmd/wio/commands/run/dependencies/flags.go

```diff
@@ -42,7 +42,7 @@ func fillPlaceholders(givenFlags, requiredFlags []string) ([]string, error) {
                 goto Continue
             }
         }
-        return nil, errors.Stringf("placeholder flag %s unfilled", required)
+        return nil, errors.Stringf("placeholder flag/definition %s unfilled in %s", required, name)
```

Comment:
it's kinda dumb to pass a variable just to return an error, if you want you can make it return `nil` if the flags are invalid and then log and error then

Reply:
True i will probably handle it differently by returning nil as you said